### PR TITLE
Post release/merge 5.2.1 into development

### PR DESCRIPTION
**Pull request by bartzet**
Include the changes from Release 5.2.1 into development.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [Next version]
 
+## [5.2.0] - 2019-05-29
```
Comment (rfreitas) [inline CHANGELOG.md]: needs to be updated (on master too I guess)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-673-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md (hunk 1 above)
Comment (bartzet) [reply CHANGELOG.md]: done and the PR to update master here: https://github.com/onfido/onfido-sdk-ui/pull/674

**Comment by rfreitas** [comment]
Travis automatic deployment: https://673-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 2]
```diff
@@ -404,8 +412,9 @@ Install with `npm install onfido-sdk-ui@0.12.0-rc.1`
 
 
 [next-version]:
-https://github.com/onfido/onfido-sdk-ui/compare/5.0.1...development
-[5.0.1]: https://github.com/onfido/onfido-sdk-ui/compare/4.0.0...5.0.0
+[5.2.1]: https://github.com/onfido/onfido-sdk-ui/compare/5.1.0...5.2.1
```
Comment (stefaniacardenas) [inline CHANGELOG.md]: How did we go from `5.1.0` to `5.2.1`? Also the development link is missing

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://673-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md (hunk 2 above)
Comment (bartzet) [reply CHANGELOG.md]: 5.2.0 got accidentally published on npm when making the release candidate. Npm doesn't allow to publish the same version again even tho I managed to unpublish it. That's why we had to bump patch.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-673-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://673-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**